examples/minimal-mdns/advertiser.cpp

```diff
@@ -190,45 +195,50 @@ OptionDef cmdLineOptionsDef[] = {
-                             "  --vendor-id <value>\n"
-                             "        Commissioning/commissionable vendor id.\n"
-                             "  --product-id <value>\n"
-                             "  -p <value>\n"
-                             "        Commissioning/commissionable product id.\n"
-                             "  --commissioning-mode <value>\n"
-                             "        Commissioning Mode (0=disabled, 1=basic, 2=enhanced).\n"
-                             "  --device-type <value>\n"
-                             "        Device type id.\n"
-                             "  --device-name <value>\n"
-                             "        Name of device.\n"
-                             "  --rotating-id <value>\n"
-                             "        Rotating Id.\n"
-                             "  --pairing-instruction <value>\n"
-                             "        Commissionable pairing instruction.\n"
-                             "  --pairing-hint <value>\n"
-                             "        Commissionable pairing hint.\n"
-                             "  --fabric-id <value>\n"
-                             "  -f <value>\n"
-                             "        Operational fabric id.\n"
-                             "  --node-id <value>\n"
-                             "  -n <value>\n"
-                             "        Operational node id.\n"
-                             "\n" };
+    "  -m <mode>\n"
+    "  --advertising-mode <mode>\n"
+    "        Advertise in this mode (operational, operational-multi-admin, commissionable-node or commissioner).\n"
+    "  --short-discriminator <value>\n"
+    "  -s <value>\n"
+    "        Commissioning/commissionable short discriminator.\n"
+    "  --long-discriminator <value>\n"
+    "  -l <value>\n"
+    "        Commissioning/commissionable long discriminator.\n"
+    "  --vendor-id <value>\n"
+    "        Commissioning/commissionable vendor id.\n"
+    "  --product-id <value>\n"
+    "  -p <value>\n"
+    "        Commissioning/commissionable product id.\n"
+    "  --commissioning-mode <value>\n"
+    "        Commissioning Mode (0=disabled, 1=basic, 2=enhanced).\n"
+    "  --device-type <value>\n"
+    "        Device type id.\n"
+    "  --device-name <value>\n"
+    "        Name of device.\n"
+    "  --rotating-id <value>\n"
+    "        Rotating Id.\n"
+    "  --pairing-instruction <value>\n"
+    "        Commissionable pairing instruction.\n"
+    "  --pairing-hint <value>\n"
+    "        Commissionable pairing hint.\n"
+    "  --fabric-id <value>\n"
+    "  -f <value>\n"
+    "        Operational fabric id.\n"
+    "  --node-id <value>\n"
+    "  -n <value>\n"
+    "        Operational node id.\n"
+    "  -t <dest>\n"
+    "  --trace-to <dest>\n"
+    "        trace to the given destination (supported: " SUPPORTED_COMMAND_LINE_TRACING_TARGETS ").\n"
```

Comment:
This line fails to compile: this seems to pull in tracing changes as well, which are not MDNS specific